runtime/jcl/j9jcl.tdf

```diff
@@ -674,3 +674,8 @@ TraceExit=Trc_JCL_java_lang_invoke_MethodHandleNatives_staticFieldBase_Exit Over
 
 TraceEntry=Trc_JCL_java_lang_invoke_MethodHandleNatives_getMemberVMInfo_Entry Overhead=1 Level=3 Template="MethodHandleNatives_getMemberVMInfo Enter: self = %p"
 TraceExit=Trc_JCL_java_lang_invoke_MethodHandleNatives_getMemberVMInfo_Exit Overhead=1 Level=3 Template="MethodHandleNatives_getMemberVMInfo Exit: result = %p"
+
+TraceEvent=Trc_JCL_initializeRequiredClasses_moduleForPackageNotFound Overhead=1 Level=3 Template="initializeRequiredClasses: module containing package %s not found"
```

Comment:
I'm thinking this and the next should be TraceException and level 1. In case something goes wrong we want to see the tracepoint.

Reply:
Good catch, thanks.

Reply:
Fixed in 648b3111e2d6.